### DISCUSSION TITLE
When will opengl support be added

### DISCUSSION THREAD

**Discussion by UnveilingCoyote** [rafaelvcaetano/melonDS-android#1072] [General]
This emulator needs opengl. It is lagging on my S22 Ultra.

**Comment by Bardock88** [comment]
> This emulator needs opengl. It is lagging on my S22 Ultra.

It is not yet defined when the OpenGL renderer will be ready, as the developer is preparing and correcting some obstacles that are in the way as you can see [here](https://github.com/rafaelvcaetano/melonDS-android#what-is-missing) ) and when it's well advanced in the functionalities it needs finalize, he it will bring news
👍.
